Composer/packages/types/src/creation.ts

```diff
@@ -22,3 +22,8 @@ export const nodeFeedKey = 'firstPartyNode';
 export const defaultFeeds = [nodeFeedKey, csharpFeedKey] as const;
 export type FeedName = typeof defaultFeeds[number];
 export type FeedType = 'npm' | 'nuget';
+
+export const webAppRuntimeKey = 'azureWebApp';
+export const functionsRuntimeKey = 'azureFunctions';
+export const availableRunTimes = [webAppRuntimeKey, functionsRuntimeKey] as const;
```

Comment:
This looks like a case for an enum.

Reply:
I have used enums for this in the past and have been told by the team to use this method instead to enforce the 'types  > enums' standard. 

@cwhitten & @hatpick I believe we discussed this in the past and this method was advised. Please correct me if I am wrong.

Reply:
I suppose at the end of the day the enum is used as a value instead of a type, however, what I'm concerned with is requiring the types package to be a prod dependency instead of a dev dependency. Maybe sense @hatpick has opinions here he can chime in with some more detail, but ideally we would be able to convert the types package into a .d.ts file -- I'm not sure if using `const` would negate that ability.

Reply:
That's a valid point @a-b-r-o-w-n, but isn't js for enum basically an IIFE that assigns enum values to the exported value as well, like this:

enum MyEnum {
  A = "A",
  B = "B"
}

becomes:
var MyEnum;
(function (MyEnum) {
    MyEnum["A"] = "A";
    MyEnum["B"] = "B";
})(MyEnum = exports.MyEnum || (exports.MyEnum = {}));

Reply:
This is true @hatpick. Do you know how @types packages handle enums?

Reply:
I think typescript inlines all enums, for example ```MyEnum.A``` is replace by ```'A'```, so there is no reference to ```MyEnum``` at runtime.